pyanaconda/core/string.py

```diff
@@ -131,3 +131,25 @@ def have_word_match(str1, str2):
     str2 = str2.lower()
 
     return all(word in str2 for word in str1_words)
+
+
+def split_in_two(value, delimiter=None):
+    """Split the given value into two strings.
```

Comment:
I would change the `value` as `string` or does it support something else than a string?

Reply:
Fixed.